### PR TITLE
Adding in two new judges - Chelsea and Michal

### PR DESCRIPTION
**Pull request by rossbruniges**

### CODE REVIEW

File: gameon/static_site/templates/static_site/judges.html [hunk 1]
```diff
@@ -111,6 +111,20 @@ <h2 class="fn exclaim">Mark Surman</h2> - <span class="org">Mozilla Foundation</
                 </article>
                 <article class="vcard">
                     <header>
+                        <h2 class="fn exclaim">Chelsea Howe</h2> - <span class="org">TinyCo</span>
+                    </header>
+                    <img class="photo" src="{{ static('base/img/judges/ChelseaHowe.jpg') }}" width="120" height="120" />
+                    <p class="whimper">Chelsea likes making games that make a difference. She’s currently a Senior Game Designer at TinyCo and works with small independent developers like Funomena and Kimochis on the side. Previously she worked at Zynga, designing and analyzing features that touched tens of millions of people, and at SuperBetter Labs she used research on positive emotion and social connection to make those touches more powerful, evocative, and meaningful. By night Chelsea designs award-winning indie games, runs the San Francisco Global Game Jam, and teaches students at California College of the Arts about playful experiences, all without a bat signal. This year, she was one of Fast Company’s 100 Most Creative People in Business and received with Microsoft WIG Rising Star award.</p>
```
Comment (andrewhayward) [inline gameon/static_site/templates/static_site/judges.html]: Do the `’` matter here? That is, as opposed to `'`, which seems to be used elsewhere in the document.

Comment (rossbruniges) [inline gameon/static_site/templates/static_site/judges.html]: We have them elsewhere on the page so I would say no. Everyone loves a bit of copy and paste

Comment (andrewhayward) [inline gameon/static_site/templates/static_site/judges.html]: Also, `and received with Microsoft WIG Rising Star award` (right at the end) doesn't seem to make sense.

File: gameon/static_site/templates/static_site/judges.html [hunk 2]
```diff
@@ -111,6 +111,20 @@ <h2 class="fn exclaim">Mark Surman</h2> - <span class="org">Mozilla Foundation</
...
+                </article>
+                <article class="vcard">
+                    <header>
+                        <h2 class="fn exclaim">Michal Budzynski</h2> - <span class="org">Mozilla</span>
+                    </header>
+                    <img class="photo" src="{{ static('base/img/judges/michalb.jpg') }}" width="120" height="120" />
+                    <p class="whimper"><a href="http://twitter.com/michalbe">Michal</a> is a Tabasco fan from Poland, addicted to TV Series and working at Mozilla on <a href="http://www.mozilla.org/firefoxos/">FirefoxOS</a>, mobile operating system formerly known as Boot2Gecko. He used to run <a href="http://openmediaweb.eu/2011/10/06/html5-games-course/">HTML5 Game Dev Workshops for W3C</a> and organizes <a href="http://ongamestart.com/">onGameStart</a>, world's first large scale HTML5 gaming conference, with <a href="http://ongamestart.us/">the first American edition planned for March 2013</a>. He currently lives in Paris, where he eats snails, frog legs and croissants.</p>
```
Comment (andrewhayward) [inline gameon/static_site/templates/static_site/judges.html]: - `...Mozilla on FirefoxOS, mobile operating system formerly known as Boot2Gecko...` doesn't really read right - needs a _the_ or equivalent.
- `...and organizes onGameStart, world's first large scale HTML5 gaming conference...` - as above.

**Comment by rossbruniges** [comment]
Fixed the above comments up

**Pull request merged**